xcodeproj/internal/xcode_schemes.bzl

```diff
@@ -246,6 +247,8 @@ def make_xcode_schemes(bazel_labels):
                 the target when executed.
             env: Optional. A `dict` of `string` values that will be set as
                 environment variables when the target is executed.
+            working_directory: Optional. A `string` that will be set as the custom
```

Comment:
Is it worth documenting what the relative path behavior is here?

Reply:
Yes, especially since off-hand I'm not sure what it is. I believe it's relative to the executable's `BUILT_PRODUCTS_DIRECTORY`, which is usually of little use to anyone.